.gitignore

```diff
@@ -10,6 +10,8 @@ telemetryCache
 quickStart.html
 .gitcommit
 media/libs/
+# Auto generated telemety defintion
+src/shared/telemetry/telemetry.ts
```

Comment:
I would suggest a `.gen.ts` filename extension, as a general practice for any generated file in any project.

Reply:
I'm not sure if typescript path mappings will allow that, if so will update

Reply:
renamed to .gen.ts